reference/sbom_naming.md

```diff
@@ -0,0 +1,31 @@
+# Best Practices for Naming and Directory Conventions for SBOMs (Software Bill of Materials) in Open Source Projects
+
+Software Bill of Materials (SBOM) plays an essential role in software supply chain transparency. Using SBOM standards such as CycloneDX and SPDX ensures interoperability, accurate dependency tracking, and efficient vulnerability management. Here are some best practices for naming and directory conventions when creating and managing SBOMs.
+
+**1. Consistent Naming Conventions:**
+
+- Use clear, descriptive names that reflect the content of the SBOM. Include the project name, version number, and the SBOM format (CycloneDX or SPDX). For example, `sbom.projectname_v1.2.3_spdx.json` or `sbom.projectname_v1.2.3_cyclonedx.xml`.
```

Comment:
I think spdx wants to be named `sbom.projectname_v1.2.3.spdx.json`, see https://spdx.github.io/spdx-spec/v2.3/conformance/#44-standard-data-format-requirements